devel/testnet/README.KR.md

```diff
@@ -83,6 +83,10 @@ docker run -v $(pwd):/agora/ bosagora/agora -c my_config.yaml
 
 위의 작업이 완료되면, 구성 파일을 다음과 같이 변경하면 됩니다.
 ```yaml
+node:
+  public_addresses:
+    - "agora://my-address.example.com:2826"
```

Comment:
Since `agora://` already have default port 2826 we don't need to specify it but it wouldn't change anything.

Reply:
Right but it's just an example, so I could use other ports like 3826, or 4826.
Thanks.